frontend/src/app/main/data/workspace/tokens/application.cljs

```diff
@@ -489,7 +489,7 @@
                                                       (some ctt/spacing-margin-keys attributes))
                                                  (ctt/any-appliable-attr? attributes (:type shape))))))
                           shape-ids (d/nilv (keys shapes)  [])
-                          any-variant? (->> shapes (some ctk/is-variant?) boolean)
+                          any-variant? (->> shapes vals (some ctk/is-variant?) boolean)
```

Comment:
This code is not strictly correct because shapes is a *seq* and not a map, `vals` and `keys` works because shapes is  a seq of MapEntry but this is a bit casual. The filter is also a bit strange here (line 486).

Do not change anything here; I just commented for information. This should be refactored in other PR